src/app/components/references/data.ts

```diff
@@ -59,7 +59,7 @@ export const measurementsFields: IFields = {
     { name: 'eligiblePopulationException', value: 'integer', description: 'The number of patients for which the measure criteria are not satisfied but who are excluded from the measure. In the measures specifications, this field is also referred to as "Denominator Exception".', notes: 'writable, optional' },
     { name: 'eligiblePopulation', value: 'integer', description: 'The total number of eligible patients as described by the measure. Must be greater than or equal to zero. In the measures specifications, this field is also referred to as "Eligible Population Denominator".', notes: 'writable, required' },
     { name: 'reportingRate', value: 'float', description: 'The reporting rate, ranging from zero to one-hundred and representing a percentage, is equal to ((performanceMet + eligiblePopulationExclusion + eligiblePopulationException + performanceNotMet) / eligiblePopulation) * 100. This is also referred to as data completeness.', notes: 'calculated by API and returned in response' },
-    { name: 'performanceRate', value: 'float', description: 'The performance rate for a single performance rate measurement, ranging from zero to one-hundred and representing a percentage, is equal to (performanceMet / (performanceMet + performanceNotMet)) * 100.', notes: `If <code>metricType = registrySingle-PerformanceRate</code> then writable and <i>required</i>.<br> If <code>metricType = singlePerformance-Rate</code>, then writeable and <i>optional</i>.` },
+    { name: 'performanceRate', value: 'float', description: 'The performance rate for a single performance rate measurement, ranging from zero to one-hundred and representing a percentage, is equal to (performanceMet / (performanceMet + performanceNotMet)) * 100.', notes: `If <code>metricType = registrySingle-PerformanceRate</code> then writable and <i>required</i>.<br> If <code>metricType = singlePerformance-Rate</code>, then calculated by API and returned in response..` },
```

Comment:
Looks like there are 2 periods at the end of this sentence: `response..`

Reply:
Literally the only way I could've messed this one up. Thanks for the catch.